Limelight/Stream/VideoDecoderRenderer.m

```diff
@@ -378,4 +372,52 @@ - (int)submitDecodeBuffer:(unsigned char *)data length:(int)length bufferType:(i
     return DR_OK;
 }
 
+- (OSStatus) decodeFrameWithSampleBuffer:(CMSampleBufferRef)sampleBuffer frameType:(int)frameType{
+    VTDecodeFrameFlags flags = kVTDecodeFrame_EnableAsynchronousDecompression;
+    
+    return VTDecompressionSessionDecodeFrameWithOutputHandler(decompressionSession, sampleBuffer, flags, NULL, ^(OSStatus status, VTDecodeInfoFlags infoFlags, CVImageBufferRef  _Nullable imageBuffer, CMTime presentationTimestamp, CMTime presentationDuration) {
+        if (status != noErr)
+        {
+            NSError *error = [NSError errorWithDomain:NSOSStatusErrorDomain code:status userInfo:nil];
+            Log(LOG_E, @"Decompression session error: %@", error);
+            LiRequestIdrFrame();
+            return;
+        }
+        
+        if (self->formatDescImageBuffer == NULL || !CMVideoFormatDescriptionMatchesImageBuffer(self->formatDescImageBuffer, imageBuffer)){
+            
+            OSStatus res = CMVideoFormatDescriptionCreateForImageBuffer(kCFAllocatorDefault, imageBuffer, &(self->formatDescImageBuffer));
+            if (res != noErr){
+                Log(LOG_E, @"Failed to create video format description from imageBuffer");
+                return;
+            }
+        }
+        
+        CMSampleBufferRef sampleBuffer;
+        CMSampleTimingInfo sampleTiming = {kCMTimeInvalid, presentationTimestamp, presentationDuration};
```

Comment:
I recommend using CACurrentMediaTime() for timing info.
In this way, the frames will be played immediately.

Reply:
I have the impression if I set it to display immediately, more jittering is generated, maybe because presentationDuration gets messed up?

Reply:
set the duration if you know the fps.

Reply:
but try it without setting the duration, I don't notice the jittering in another project.